src/components/10-atoms/spinner/story.js

```diff
@@ -0,0 +1,41 @@
+import { boolean, radios, withKnobs } from '@storybook/addon-knobs';
+import { html, render } from 'lit-html';
+import './index';
+import readme from './README.md';
+import changelog from './CHANGELOG.md';
+
+export default {
+  title: 'Components/Spinner',
+  decorators: [withKnobs],
+  parameters: {
+    readme,
+    usage: {
+      componentName: 'spinner',
+    },
+    changelog,
+  },
+};
+
+const invertedBgs = {
```

Comment:
What does that mean? We should only use abbrevations if they are "common". So better write it out.